pkg/bookmarks/bookmarks.go

```diff
@@ -72,3 +89,22 @@ func ReadAll(path string) (map[string]Bookmark, error) {
 
 	return results, nil
 }
+
+type ErrNonExistingBookmark string
```

Comment:
i see your point about not using a new pattern to keep things consistent. i would however like to point the advantage of this pattern - having an explicit typed error means that in the unit tests, i can assert equality on _types_ of error rather than doing a vague string assertion. example here -

`assert.Equal(t, ErrNonExistingBookmark("bar"), err)`